deps/build.jl

```diff
@@ -19,7 +19,7 @@ if is_apple()
 end
 
 version = "2.0.2"
-win_version = "2.0.2" # 2.0.* is not yet working on windows
+win_version = "2.0.2"
```

Comment:
Why remove this comment when the current build script still does not support windows?

Reply:
SCS 2.0.2 works fine with windows, the binaries need to be updated for SCS.jl. I updated the comment.

Reply:
Ok, good to clarify.